spec/fixtures/files/datahashes/core/collectionobject1.json

```diff
@@ -52,10 +49,10 @@
     "measuredPart": "image-size",
     "dimensionSummary": "charger ya samsung",
     "dimension": "circumference^^",
-    "measuredByPerson": "afa^^",
+    "measuredByPerson": "afa",
     "measuredByOrganization": "pandemic",
     "measurementMethod": "standard_mesh_screen^^standard_mesh_screen",
-    "value": "32;",
```

Comment:
; indicates there are two Dimensions > Part groups, but there is only one.

^^ indicates the two Measurement > Dimension values within your Dimensions > Part group.

View [anthro collectionobject CollectionSpace record](https://anthro.dev.collectionspace.org/cspace/anthro/record/collectionobject/db433216-166e-4fe7-b795) and JSON file for example of multiple Dimensions > Part groups, each including multiple Measurement > Dimension subgroups